test/async-hooks/init-hooks.js

```diff
@@ -65,50 +65,55 @@ class ActivityCollector {
     }
 
     const violations = [];
+    let tempActivityString;
     function v(msg) { violations.push(msg); }
     for (const a of this._activities.values()) {
+      tempActivityString = activityString(a);
       if (types != null && !types.includes(a.type)) continue;
 
       if (a.init && a.init.length > 1) {
-        v('Activity inited twice\n' + activityString(a) +
-          '\nExpected "init" to be called at most once');
+        v(`Activity inited twice
+          ${tempActivityString}
+          Expected "init" to be called at most once`);
```

Comment:
Please prevent use multiline template strings. This adds lots and lots of whitespace infront of line 2 + 3. Instead, keep the `+` for each new line and use `\n` if necessary..

Reply:
@BridgeAR Thanks for the feedback. Removed the multiline template strings and went back to using `+` and `\n`.